plugin/kubernetes/pvc.go

```diff
@@ -265,6 +266,42 @@ func (v *pvc) VolumeStream(ctx context.Context, path string) (io.ReadCloser, err
 	return plugin.CleanupReader{ReadCloser: output, Cleanup: delete}, nil
 }
 
+func (v *pvc) VolumeDelete(ctx context.Context, path string) (bool, error) {
+	// Create a container that mounts a pvc and inspects it. Run rm -rf on it.
```

Comment:
This is also almost an exact copy of VolumeList. Only difference is the command run and that VolumeList uses the output in its return value.